src/main/java/com/dreammaster/recipes/RecipeRemover.java

```diff
@@ -53,7 +53,13 @@ private static void stopBuffering() {
         int i = tList.size();
         tList.removeIf(r -> {
             ItemStack rCopy = r.getRecipeOutput();
-            if (rCopy == null) return false; // ????????????????????
+            if (rCopy == null) {
+                return false;
+            }
+            if (rCopy.getItem() == null) {
+                System.out.println("Someone is adding recipes with null items!");
```

Comment:
Use the logger like below, not system.out:
```suggestion
                MainRegistry.Logger.warn("Someone is adding recipes with null items!");
```

Reply:
might also be a good idea to throw a stacktrace in the log to see where the recipe is

Reply:
Should I log it in warn (or other log level) or directly `Thread.dumpStack()`?

Reply:
Log it in warn